doc/userdoc/installation/cmake_options.rst

```diff
@@ -28,70 +28,116 @@ Options for configuring NEST
-                                                 library, set install path.[default=ON]
-    -Dwith-readline=[OFF|ON|</path/to/readline>] Find a GNU Readline library. To set
-                                                 a specific library, set install path.
-                                                 [default=ON]
-    -Dwith-ltdl=[OFF|ON|</path/to/ltdl>]         Find an ltdl library. To set a specific
-                                                 ltdl, set install path. NEST uses the
-                                                 ltdl for dynamic loading of external
-                                                 user modules. [default=ON]
-    -Dwith-python=[OFF|ON]                       Build PyNEST. [default=ON]
-    -Dcythonize-pynest=[OFF|ON]                  Use Cython to cythonize pynestkernel.pyx.
-                                                 If OFF, PyNEST has to be build from
-                                                 a pre-cythonized pynestkernel.pyx.
-                                                 [default=ON]
-
-Change compilation behavior::
-
-    -Dstatic-libraries=[OFF|ON]     Build static executable and libraries. [default=OFF]
-    -Dwith-optimize=[OFF|ON|<list;of;flags>]       Enable user defined optimizations. Separate
-                                                   multiple flags by ';'.
-                                                   [default OFF, when ON, defaults to '-O3']
-    -Dwith-warning=[OFF|ON|<list;of;flags>]        Enable user defined warnings. Separate
-                                                   multiple flags by ';'.
-                                                   [default ON, when ON, defaults to '-Wall']
-    -Dwith-debug=[OFF|ON|<list;of;flags>]          Enable user defined debug flags. Separate
-                                                   multiple flags by ';'.
-                                                   [default OFF, when ON, defaults to '-g']
-    -Dwith-intel-compiler-flags=[<list;of;flags>]  User defined flags for the Intel compiler.
-                                                   Separate multiple flags by ';'.
-                                                   [defaults to '-fp-model strict']
-    -Dwith-libraries=<list;of;libraries>           Link additional libraries. Give full path.
-                                                   Separate multiple libraries by ';'.
-                                                   [default OFF]
-    -Dwith-includes=<list;of;includes>             Add additional include paths. Give full
-                                                   path without '-I'. Separate multiple include
-                                                   paths by ';'. [default OFF]
-    -Dwith-defines=<list;of;defines>               Additional defines, e.g. '-DXYZ=1'.
-                                                   Separate multiple defines by ';'. [default OFF]
+
+**Change parallelization scheme:**
```

Comment:
Should these all be headings (which level?) instead of explicitly bold-faced? I'd remove the colon at the end.

And maybe "Select" instead of "Change"?